libs/dh/metering-point/feature-search/src/lib/form/dh-metering-point-search-form.component.html

```diff
@@ -0,0 +1,50 @@
+<!--
+@license
+Copyright 2021 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<form
+  *transloco="let transloco; read: 'meteringPoint.search'"
+  name="search-form"
+>
+  <watt-form-field size="large">
+    <watt-label class="watt-text-m">{{ transloco('searchLabel') }}</watt-label>
+    <watt-icon name="search" wattPrefix></watt-icon>
+    <input
+      #searchInput
+      wattInput
+      type="text"
+      [placeholder]="transloco('searchPlaceholder')"
+      [formControl]="searchControl"
+      inputmode="numeric"
+    />
+    <watt-error *ngIf="searchControl.errors?.invalidMeteringPointId">
+      {{ transloco('searchInvalidLength') }}
+    </watt-error>
+    <watt-icon-button
+      icon="close"
+      wattSuffix
+      role="button"
```

Comment:
question:nitpick: isn't this unnecessary? The `watt-icon-button` is rendering a button internally.

Reply:
To resolve this I've added a label for the icon button, to make it possible to set the aria-label of the button rendered. Otherwise the button will be named by the icon, in this case close which doesn't really makes sense.